src/components/MainPanel/ConversationRow.scss

```
@@ -0,0 +1,149 @@
+    border: 1px solid #3a3a3a;
+  }
+
+  &.tr {
+    background: rgba(16, 163, 127, 0.2);
+    color: #10a37f;
+    border: 1px solid rgba(16, 163, 127, 0.4);
+  }
+}
+
+.row-text {
+  word-break: break-word;
```

Stylelint (17.7.0)

[error] Deprecated keyword "break-word" for property "word-break" (declaration-property-value-keyword-no-deprecated)

(declaration-property-value-keyword-no-deprecated)